patches/chrome-BUILD.gn.patch

```diff
@@ -58,34 +58,34 @@ index 36b6fc1d94957475528e7a1de2ec97cfca682682..e795f4e983e78ba7f8f32d101af8b247
    compile_entitlements("entitlements") {
      entitlements_templates = [ "app/app-entitlements.plist" ]
 +    entitlements_templates += brave_entitlements_templates
```

Comment:
we should also use `include_branded_entitlements` to selectively exclude our entitlements for locally signed builds

Reply:
Is this for a follow up?

Reply:
yes

Reply:
Filed https://github.com/brave/brave-browser/issues/26651